### PR TITLE
little improvements

### PR DESCRIPTION
**Pull request by zerai**

### CODE REVIEW

File: src/Badge/Service/ClientStrategy.php [hunk 1]
```diff
@@ -13,9 +13,9 @@
 
 class ClientStrategy
 {
-    private const GITHUB_SOURCE = 'github.com';
+    //private const GITHUB_SOURCE = 'github.com';
```
Comment (AlessandroMinoccheri) [inline src/Badge/Service/ClientStrategy.php:16]: we should remove commented lines if they are not useful.

File: src/Badge/Service/ClientStrategy.php [hunk 2]
```diff
@@ -28,72 +28,65 @@ public function __construct(GithubClient $githubClient, BitbucketClient $bitbuck
...
+            $repoGitHubData = $repoApi->show($username, $repositoryName);
+            if (!$this->isValidGithubRepository($repoGitHubData)) {
+                throw new RepositoryDataNotValid('Repository data not valid: '.(string) \json_encode($repoGitHubData));
+            }
+
+            return (string) $repoGitHubData['default_branch'];
         }
 
-        return $defaultBranch;
+        if ($repository->isBitBucket()) {
```
Comment (AlessandroMinoccheri) [inline src/Badge/Service/ClientStrategy.php:55]: I think that correct name is Bitbucket and not BitBucket

Comment (zerai) [reply src/Badge/Service/ClientStrategy.php:55]: ops!! I'll fix..

File: src/Badge/Service/ClientStrategy.php [hunk 3]
```diff
@@ -28,72 +26,65 @@ public function __construct(GithubClient $githubClient, BitbucketClient $bitbuck
         $this->bitbucketClient = $bitbucketClient;
     }
 
+    /**
+     * @throws SourceClientNotFound
+     * @throws \Http\Client\Exception
```
Comment (JellyBellyDev) [inline src/Badge/Service/ClientStrategy.php:31]: please use short annotation ad add import for the class. Thanks!

Comment (zerai) [reply src/Badge/Service/ClientStrategy.php]: Done..

**Comment by JellyBellyDev** [comment]
Luckily I modified the CircleCI configurations to trigger the suite even for forked repo! :D
@zerai please fix the pipeline! ;)

**Pull request merged**